web/src/components/moreButton.js

```diff
@@ -11,7 +11,7 @@ export default function MoreButton({ text, isDisabled, isLeft }) {
     } ${isLeft ? leftStyle : rightStyle}`;
 
     return (
-        <button className={finalStyles} disabled={isDisabled} type="button">
+        <button className={finalStyles} isDisabled={isDisabled} type="button">
```

Comment:
should this attribute be named `isDisabled`? [mdn docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-disabled)